docs/build-hibench.md

```diff
@@ -28,7 +28,7 @@ Because some Maven plugins cannot support Scala version perfectly, there are som
 
 
 ### Specify Spark Version ###
-To specify the spark version, use -Dspark=xxx(1.6, 2.0, 2.1 or 2.2). By default, it builds for spark 2.0
+To specify the spark version, use -Dspark=xxx(1.6, 2.0, 2.1, 2.2 or 2.4). By default, it builds for spark 2.0
```

Comment:
Actually this Spark 2.4 support doesn't include `SQL` module? 
This is the only main remaining concern for this patch, see
https://github.com/Intel-bigdata/HiBench/pull/586#discussion_r310899252
I think we can drop Spark 1.6 support and modify the SQL module code to support 2.4 in HiBench 8.0, whoever needs 1.6 can go to HiBench 7.0. @carsonwang